packages/mask/src/plugins/Collectible/SNSAdaptor/TokenTab.tsx

```diff
@@ -156,7 +164,14 @@ export function TokenTab(props: TokenTabProps) {
                 <Box className={classes.chain_row}>
                     <Typography variant="body2">{t('plugin_collectible_contract_address')}</Typography>
                     <Link
-                        href={explorerResolver.addressLink(ChainId.Mainnet, token?.contractAddress ?? '')}
+                        href={
+                            provider === SourceType.MagicEden
+                                ? solanaExplorerResolver.addressLink(
```

Comment:
```ts
const { Others } = useWeb3State()

Others.explorerResolver.addressLink()
```

Reply:
The result will be incorrect when `currentPluginId` is `EVM`.

Reply:
Need to wrap the whole card inside a PluginContext